pylint/checkers/stdlib.py

```diff
@@ -253,6 +257,12 @@
             "unittest.TestLoader.loadTestsFromTestCase",
             "unittest.TestLoader.getTestCaseNames",
         },
+        (3, 12, 0): {
+            "builtins.bool.__invert__",
```

Comment:
This one doesn't have any effect because `~` is currently uninferable, but it doesn't hurt to have this here in case we fix it in astroid later.